lib/ueberauth/strategy/google.ex

```diff
@@ -133,6 +148,12 @@ defmodule Ueberauth.Strategy.Google do
     end
   end
 
+  defp put_user(conn, user) do
+    token = %OAuth2.AccessToken{}
```

Comment:
Why the empty token here? Seems like having the user without the token is less than ideal since you wouldn't be able to use the token to make server side calls.